### ISSUE TITLE
We should have a white mode 🌝

### ISSUE THREAD

**Issue by Toto12345678** [yellowme/slack-poll-web#5]

**Comment by rejonpardenilla** [comment]
I agree. Give me some time to add the light theme to the design on Zeplin 😄

**Comment by rejonpardenilla** [comment]
You can check the design here if you want: https://zpl.io/blxQd8X

**Comment by rejonpardenilla** [comment]
![Desktop HD Copy](https://user-images.githubusercontent.com/22482675/73956512-a9590a80-48ca-11ea-822b-82c506389cb3.png)
@Toto12345678 what do you think?

**Comment by rejonpardenilla** [comment]
🤔 Where do you think we should put the button to toggle the themes?

**Comment by Toto12345678** [comment]
Check this Day/Night implementation using vue.js https://www.mynotepaper.com/create-multiple-themes-in-vuejs

**Comment by rejonpardenilla** [comment]
I updated both zeplin views to look like this:
![image](https://user-images.githubusercontent.com/22482675/74613616-8f42d780-50d5-11ea-8208-37f2869beaf8.png)
![image](https://user-images.githubusercontent.com/22482675/74613632-a1bd1100-50d5-11ea-99c9-2f4dbe6fc282.png)
